### ISSUE TITLE
failure takePicture Error: Request failed with status code 500

### ISSUE THREAD

**Issue by junior466** [koush/scrypted#48]
I really want to get Scrypted working in my environment (so thanks a lot by the way!) but I noticed that I am not receiving alerts or having footage recorded. It did work once yesterday but have since stopped.

I do see this in the logs and can also provide more info if you tell me where to look.

        Nov 14 13:33:35 VM_ npx[979]: e HomeKit unhandledRejection Error: @scrypted/homekit:host: host:@scrypted/amcrest: Request failed with status code 500
        Nov 14 13:33:51 VM_ npx[979]: failure takePicture Error: Request failed with status code 500
        Nov 14 13:33:51 VM_ npx[979]:     at createError (/node_modules/axios/lib/core/createError.js:16:15)
        Nov 14 13:33:51 VM_ npx[979]:     at settle (/node_modules/axios/lib/core/settle.js:17:12)
        Nov 14 13:33:51 VM_ npx[979]:     at IncomingMessage.<anonymous> (/node_modules/axios/lib/adapters/http.js:269:11)
        Nov 14 13:33:51 VM_ npx[979]:     at IncomingMessage.emit (node:events:402:35)
        Nov 14 13:33:51 VM_ npx[979]:     at IncomingMessage.emit (node:domain:475:12)
        Nov 14 13:33:51 VM_ npx[979]:     at endReadableNT (node:internal/streams/readable:1343:12)
        Nov 14 13:33:51 VM_ npx[979]:     at processTicksAndRejections (node:internal/process/task_queues:83:21) {
        Nov 14 13:33:51 VM_ npx[979]:   config: {
        Nov 14 13:33:51 VM_ npx[979]:     url: 'http://192.168.141.17:80/cgi-bin/snapshot.cgi',
        Nov 14 13:33:51 VM_ npx[979]:     method: 'get',
        Nov 14 13:33:51 VM_ npx[979]:     headers: {
        Nov 14 13:33:51 VM_ npx[979]:       Accept: 'application/json, text/plain, */*',
        Nov 14 13:33:51 VM_ npx[979]:       authorization: 'Digest username="admin", realm="Login to AMC0658A9030EBD927", nonce="202890372", uri="/cgi-bin/snapshot.cgi", qop="auth", algorithm="MD5", response="7ec1701b9bcbe0a06ad864b1f0556adc", nc="00000029", cnonce="304a3ac08291d10a7e46b08ded5f1d59d15f4b30b1e89ba5"',
        Nov 14 13:33:51 VM_ npx[979]:       'User-Agent': 'axios/0.21.4'

**Comment by koush** [comment]
your camera or or crashed and is returning 500s. amcrest I'm guessing. needs to be rebooted.

**Comment by bobrik** [comment]
It seems that Amcrest doesn't allow you to have snapshots faster than you automatically take them. I enabled my camera to take a snapshot every 60s and it was mostly giving `500 Internal Server Error` after 15s when I requested them manually (or of scrypted requested them for homekit). Rebooting the camera did nothing. Disabling storage on NFS also did nothing.

Asking Amcrest to take snapshots every 1s allowed my curls to succeed every time. 

<img width="714" alt="image" src="https://github.com/koush/scrypted/assets/89186/4d66eb79-0dcf-4b78-b3aa-849d6999cc53">

Naturally, it's not compatible with saving minutely snapshots from the camera on NFS as I initially hoped.

It doesn't make any logical sense, but it works.
